superset/viz.py

```diff
@@ -185,6 +185,17 @@ def get_fillna_for_columns(self, columns=None):
         }
         return fillna
 
+    def get_samples(self):
```

Comment:
Should we add a sort by random here? I guess it would be hard since it's database dependent...

Reply:
Yup that would be database dependent and could be very expansive...